planet/models.py

```diff
@@ -151,15 +142,13 @@ def size(self):
     @property
     def num_bytes_downloaded(self):
         '''The number of bytes downloaded.
-
         :returns: number of bytes downloaded
         :rtype: int
         '''
         return self.response.num_bytes_downloaded
 
     def last_modified(self):
         '''Read the last-modified header as a datetime, if present.
-
```

Comment:
Is there a reason these empty lines were removed? Their presence increases readability of the docstrings. I suggest undoing all the 'remove empty line' changes